<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choice page</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>470</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 158, 243);</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>-20</y>
     <width>220</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NSimSun</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Student</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>290</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(191, 148, 255);
border-radius:10px;
	color: rgb(255, 255, 255);
}
QPushButton:hover{
color: rgb(255, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Play Quiz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>290</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(191, 148, 255);
border-radius:10px;
	color: rgb(255, 255, 255);
}
QPushButton:hover{
color: rgb(255, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Score card</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>290</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(191, 148, 255);
border-radius:10px;
	color: rgb(255, 255, 255);
}
QPushButton:hover{
color: rgb(255, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>290</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(191, 148, 255);
border-radius:10px;
	color: rgb(255, 255, 255);
}
QPushButton:hover{
color: rgb(255, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Sign Out</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
